packages/terra-button-group/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+*Fixed
+  *Fixed Terra- Button Group Accessiblity Issues.
```

Comment:
Needs a correction ! Alignment is not matching with existing logs

```suggestion

* Fixed
  * Fixed Terra- Button Group Accessibility Issues.
```

Reply:
I have made change as per Suggestion.